docs/compose-file.md

```diff
@@ -552,7 +552,7 @@ limit as an integer or soft/hard limits as a mapping.
         soft: 20000
         hard: 40000
 
-### volumes, volume\_driver
+### volumes
 
 Mount paths or named volumes, optionally specifying a path on the host machine
 (`HOST:CONTAINER`), or an access mode (`HOST:CONTAINER:ro`). Named volumes can
```

Comment:
Hm, yeah. There's also no top-level volume key in v1, so I think that part needs a small rewrite.